### ISSUE TITLE
JIT engine does not fuse elementwise on 1.7.1 while 1.6.0 is fine

### ISSUE THREAD

**Issue by franksun007** [pytorch/pytorch#52169]
## 🐛 Bug

We used the autograd profiler to discover the problem. Our JIT function is pretty simple: 
```
@torch.jit.script
def fake_q(x, scale, dtype_min: float, dtype_max: float):
    f = x / scale
    f = torch.round(f)
    f = f.clamp(dtype_min, dtype_max)
    f = f * scale
    return f
```
Through the autograd profiler, the above function is marked as a `FushionGroup` in PyTorch 1.6.0, but as a `fallback_function` when using 1.7.0 and 1.7.1. The performance on 1.7.1 is worst than 1.6.0 as well. 

What should I do to fuse the above elementwise operations?  

## To Reproduce

1. Copy paste the above function, and run through the autograd profiler. 
1. You will observe the trace treat the above function as a `FushionGroup` in PyTorch 1.6.0 while `fallback_function` in PyTorch 1.7.1

## Expected behavior

I should not expect a performance degradation when using 1.7.1

## Environment

```
Collecting environment information...
PyTorch version: 1.6.0+cu101
Is debug build: False
CUDA used to build PyTorch: 10.1
ROCM used to build PyTorch: N/A

OS: Ubuntu 18.04.3 LTS (x86_64)
GCC version: (Ubuntu 7.4.0-1ubuntu1~18.04.1) 7.4.0
Clang version: Could not collect
CMake version: version 3.10.2

Python version: 3.6 (64-bit runtime)
Is CUDA available: True
CUDA runtime version: Could not collect
GPU models and configuration: GPU 0: Tesla V100-SXM2-32GB
Nvidia driver version: 418.40.04
cuDNN version: /usr/lib/x86_64-linux-gnu/libcudnn.so.7.4.2
HIP runtime version: N/A
MIOpen runtime version: N/A

Versions of relevant libraries:
[pip3] numpy==1.17.5
[pip3] torch==1.6.0+cu101
[pip3] torchaudio==0.7.2
[pip3] torchvision==0.7.0+cu101
[conda] numpy                     1.17.5                   pypi_0    pypi
[conda] torch                     1.6.0+cu101              pypi_0    pypi
[conda] torchaudio                0.7.2                    pypi_0    pypi
[conda] torchvision               0.7.0+cu101              pypi_0    pypi
```

 - PyTorch Version (e.g., 1.0): 1.6.0, and 1.7.1
 - OS (e.g., Linux): Ubuntu
 - How you installed PyTorch (`conda`, `pip`, source): pip
 - Build command you used (if compiling from source):
 - Python version: 3.6
 - CUDA/cuDNN version: 10.1
 - GPU models and configuration: V100
 - Any other relevant information: /

## Additional context

The above function is not a serious problem, but we have another function that is really performance critical and not getting fused. 

cc @gmagogsfm

**Comment by eellison** [comment]
Hi @franksun007, I tried out your example and it was getting fused (at least on master). Could you try posting a more complete repro ?

**Comment by franksun007** [comment]
Hi, it seems like this issue is resolved in 1.8. Closing. Thanks
